data/topic-schema/src/main/kotlin/net/corda/schema/Schemas.kt

```diff
@@ -129,6 +129,7 @@ class Schemas {
             const val P2P_IN_TOPIC = "p2p.in"
             const val P2P_HOSTED_IDENTITIES_TOPIC = "p2p.hosted.identities"
             const val P2P_MGM_ALLOWED_CLIENT_CERTIFICATE_SUBJECTS = "p2p.mgm.allowed.client.certificate.subjects"
+            const val P2P_ALL_ALLOWED_CLIENT_CERTIFICATE_SUBJECTS = "p2p.all.allowed.client.certificate.subjects"
```

Comment:
how about the following (and drag it slightly below with the gateway topics)?
```suggestion
            const val GATEWAY_ALLOWED_CLIENT_CERTIFICATE_SUBJECTS = "gateway.allowed.client.certificate.subjects"
```
A lot of the topics where we forward data to the gateway are prefixed with that at the moment, so it'd be nice to keep it consistent.